python/ray/serve/_private/http_proxy.py

```diff
@@ -417,6 +440,12 @@ async def __call__(self, scope, receive, send):
                 }
             )
 
+        # Decrement the ongoing request counter and drop the dummy object reference
```

Comment:
Could you log whenever the dummy object is either put or removed from the object store?

Reply:
I think this may get noisy -- how about logging this information only when the proxy is in the draining (currently named inactive) state?

Reply:
Oo yea I think that will be the case logs gets noisy. Will keep this out. Already have [logs](https://github.com/ray-project/ray/pull/36652/files#diff-c9d171f0ce6d67a807f1fa36aeba6eccbfff961279c09d25f0ad178c5ff58d04R752) for active state changing, will rename all of them to draining :)